woocommerce/plugin/wp-content/plugins/indodana-payment/indodana-payment-gateway.php

```diff
@@ -238,6 +238,21 @@ public function getTotalTaxAmount($cart)
     return (float) $cart->get_total_tax();
   }
 
+  public function getTotalAdminFeeAmount($cart)
+  {
+    return (float) 0;
```

Comment:
Why are we hardcoding this? is it because we just want to support it so it doesn't break the code?